docs/proposals/project-repos-and-clusters.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Neat-enhancement-idea
+authors:
+- "@alexmt"
+  sponsors:
+- TBD
+  reviewers:
+- "@jessesuen"
+- TBD
+  approvers:
+- "@jessesuen"
+- TBD
+
+creation-date: 2020-04-19
+last-updated: 2020-04-19
+---
+
+# Neat Enhancement Idea
+
+Support project scoped Repositories and Clusters to enable self-service end-users onboarding.
+
+## Summary
+
+The Argo CD has two type of users:
+
+* Administrators who configure the Argo CD and manage the Argo CD projects.
+* Developers who use Argo CD to manage resources in the Kubernetes clusters.
+
+These two roles enable sharing on the Argo CD instance in a multi-tenant environment. Typically the developer
+requests a new project from an administrator. The administrator creates the project, defines which repositories
+can and clusters can be used within the project which concludes the onboarding.
+
+The problem is that list of repositories and clusters often not known during the onboarding process. Developers get
```

Comment:
grammar: list of repositories and clusters *are* often not

Reply:
fixed